### PR TITLE
SF3.x compatibility + tests

### PR DESCRIPTION
**Pull request by mgiraud**

### CODE REVIEW

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -1,13 +1,9 @@
 <?php
-
```
Comment (tuxone) [inline DependencyInjection/Configuration.php]: Hi @mgiraud! Thank you very much for this PR! I was just wondering, why did you remove all these blank lines?

**Comment by mgiraud** [comment]
Any chance you merge this soon ?
